docs/annexes/spdx-lite.md

```diff
@@ -121,7 +121,7 @@ Additionally:
 - Mandatory
     1. created
     1. createdBy (may be multiple), SHOULD be objects of type /Core/Agent
-    1. specVersion, MUST be a fixed string, “3.0.1”.
+    1. specVersion, MUST be a fixed string, “3.0.*” - where * is any supported dot version of the SPDX specification.
```

Comment:
I'm not sure which one is more formal between "patch release", "point release", and "dot release".

Reply:
I agree with the "patch version".